submissions/iakow/OOP-exercise/index.js

```diff
@@ -0,0 +1,87 @@
+class Inhabitant {
+  constructor({ species, name, gender, legs, hands, say }) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.legs = legs;
+    this.hands = hands;
```

Comment:
Some of our inhabitants do not have hands by design. The beauty of OOP/SOLID is that every base class handles props shared by its child classes, and every child class introduces props unique to that child class.
This way we make scalable and maintainable code.

It is important that every class handles its properties. I.e. a base class should know nothing about props unique to its child classes and therefore may not refer to them.